src/pages/Productdetail/ProductDetailOption.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+class ProductDetailOption extends React.Component {
+  render() {
+    const { discount_price } = this.props.detailData;
+
+    return (
+      <section className="productDetailInfo">
+        <table className="productDetailOption">
+          <tbody>
+            <tr className="productDetailOptionTable">
+              <td className="optionName">
+                [오리지널] 텐미닛 플래너 31DAYS - 화이트
+              </td>
+              <td>
+                <span className="quantitiyBlock">
+                  <input
+                    className="quantity"
+                    value="1"
+                    type="text"
+                    randonly="randonly"
```

Comment:
readonly 속성을 부여하실려고 하신건가요?
오타 있는 것 같습니다! 실제로 제대로 동작되는지 확인해보세요!
그리고 readonly는 true or false의 boolean값입니다 따라서 아래처럼 
```suggestion
                    randonly
```
속성을 부여해주기만 하면 true로 처리됩니다!

Reply:
넹 readonly를 주고싶었습니다! 오탈자 확인하겠습니다. 원하는 속성이 disabled라서 disabled로 지정했습니다! 감사합니다.